lib/PrometheusPlugin.js

```diff
@@ -74,33 +104,68 @@ class PrometheusPlugin {
    * @param {KuzzlePluginContext} context
    */
   async init(config, context) {
-    this.config = { ...this.config, ...config };
+    Object.assign(this.config.labels, config.labels || {});
```

Comment:
(nitpicking) you do not have to test if config.labels is set, the behavior of `Object.assign` is to do nothing with undefined, null or non-iterable arguments.

Reply:
:+1: